Console/Console/Program.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.IO;
+using System;
+
+while (true)
+{
+try
+{
+        bool XP;
+        int X;
+        int Y;
+        do
+        {
+            Console.WriteLine("Input integer X");
+
+            XP = int.TryParse(Console.ReadLine(), out X);
+        } while (!XP);
+        do
+        {
+            Console.WriteLine("Input integer Y");            
+            XP = int.TryParse(Console.ReadLine(), out Y);
+        } while (!XP);
+
+    if (X != Y)
```

Comment:
you do not need if/else, if X == Y your loop will run only once and return correct sum